### DISCUSSION TITLE
Is flagged as a trojan on Virus Total

### DISCUSSION THREAD

**Discussion by Greenman23** [X2CommunityCore/xcom2-launcher#372] [General]
Version 1.5:

When I uploaded this file on virus two security vendor viewed as malicious.

Link: https://www.virustotal.com/gui/file/b44760eed4e505754924acf94f1feeec1868fa593d76bcc8e4c56ba093fc2b5c

**Comment by robojumper** [comment]
For a niche application some false positives from small anti-virus vendors are expected. Considering the file is two years old, I think a lot more vendors would have detected this as a virus if it actually were malicious.

If any of the bigger vendors flagged the file we'd probably want to contact them about it but I've honestly never even heard about MaxSecure or VirIT.
